Readme.md

```diff
@@ -1,5 +1,8 @@
 # Web server for image classification
 
+## Documentation
```

Comment:
I am accepting this, for now. 
However, it is not recommended to write documentation in external repositories (as if I merge this here it will become an external repo). Unfortunately, there is no method to create a pull request for the wiki pages, so it is usually better to handle this with documentation tools (_e.g._, readthedocs). 
In this case, we can consider this project to be in the early development stage, so I am accepting this change. I just wanted to specify for future encounters :wink: